x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_list/credentials_list.test.tsx

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import '../../../../__mocks__/kea.mock';
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { useValues, useActions } from 'kea';
+
+import { CredentialsList } from './credentials_list';
+import { EuiBasicTable, EuiCopy } from '@elastic/eui';
+import { IApiToken } from '../types';
+import { ApiTokenTypes } from '../constants';
+
+describe('Credentials', () => {
+  const apiToken: IApiToken = {
+    name: '',
+    type: ApiTokenTypes.Private,
+    read: true,
+    write: true,
+    access_all_engines: true,
+    key: 'abc-1234',
+  };
+
+  const mockKea = ({ values = {}, actions = {} }) => {
```

Comment:
Would love to simplify this again a la `credentials.text.tsx` - here's a diff to cherry-pick if you'd like it!

https://github.com/constancecchen/kibana/commit/4b42c317311d70a7bd81a4c2d56e8f77d24f2fbf

Reply:
Ah yeah, sorry, I should have switched that over after your last comment. Thanks.